spec/Body/Body.vspec

```diff
@@ -161,11 +161,57 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights.
+#include StaticLights.vspec Lights.Beam
+
+Lights.Running:
+  type: branch
+  description: Running lights.
+#include StaticLights.vspec Lights.Running
+
+Lights.Backup:
+  type: branch
+  description: Backup lights.
+#include StaticLights.vspec Lights.Backup
+
+Lights.Parking:
+  type: branch
+  description: Parking lights.
+#include StaticLights.vspec Lights.Parking
+
+Lights.Fog:
+  type: branch
+  instances: ["Rear","Front"]
+  description: Fog lights.
+#include StaticLights.vspec Lights.Fog
+
+Lights.LicensePlate:
+  type: branch
+  instances: ["Rear","Front"]
+  description: License plate lights.
+#include StaticLights.vspec Lights.LicensePlate
+
+Lights.Brake:
+  type: branch
+  instance: ["Left","Right","Top"]
+#include BrakeLights.vspec Lights.Brake
+
+Lights.Hazard:
+  type: branch
+  description: Hazard lights.
+#include SignalingLights.vspec Lights.Hazard
 
+Lights.Indicator:
+  type: branch
+  instances: ["Left","Right"]
+  description: Indicator lights.
```

Comment:
To avoid misunderstandings, would it be an idea to include alternative terms in the description like:

`description: Indicator (turn signal) lights.`

 Or possibly use the term DirectionIndicator for the signal as used in e.g. https://unece.org/fileadmin/DAM/trans/main/wp29/wp29regs/2015/R048r12e.pdf

I actaully do not know if the terms we are using for the signal names are based on any standard, to me they seem to be a bit random.